.github/workflows/mvnw.yml

```diff
@@ -0,0 +1,25 @@
+name: Java CI
+
+#on:
+#  schedule:
+#    - cron: '0 16 * * *'
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      # Step that does that actual cache save and restore
+      - uses: actions/cache@v1
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+           ${{ runner.os }}-maven-
+      - name: Build with mvnw
+        run: ./mvnw clean install -DskipTests -Ddockstore-core.version=1.12.0-alpha.1-SNAPSHOT -B -ntp -U
```

Comment:
1. Don't we want to run the tests, or is this just a compiler check?
2. Not 100% sold on this idea, but what if the dockstore-core.version value came from an environment variable?

Reply:
1. Probably going to need to ask Walt about this, but it was my assumption that compiler check would've caught most if not all the issues that appeared during release 
2. Hmm...not sure. I think for GitHub admins it would be easier with environment variable since you don't have to do a PR (which avoids wait time and such). But for non-admins, it's either much more difficult or impossible

Reply:
Might as well run unit tests to avoid using the really crummy builds